pkg/infra/kubernetes/exec_unit.go

```diff
@@ -165,6 +169,13 @@ func (unit *HelmExecUnit) transformDeployment() ([]Value, error) {
 	deployment.Spec.Selector.MatchLabels["execUnit"] = unit.Name
 
 	deployment.Spec.Template.Spec.ServiceAccountName = unit.getServiceAccountName()
+
+	k8sCfg := cfg.GetExecutionUnitParamsAsKubernetes()
+	if k8sCfg.Limits.Cpu != 0 {
+		container.Resources.Limits = mapOrNew(container.Resources.Limits)
+		container.Resources.Limits[corev1.ResourceCPU] = *k8s_resource.NewDecimalQuantity(*inf.NewDec(int64(k8sCfg.Limits.Cpu), 0), k8s_resource.DecimalSI)
```

Comment:
just a note, cpu limit is in millicores so we should make sure we align config with that

Reply:
er, I'm not sure what to do with that statement

Reply:
im guessing your statement converts to millicores, just not sure exactly what that statement does. but just saying our config is an int so 1 isnt 1 cpu its .0001 cpu or whatever

Reply:
I think it just takes whatever int was in the yaml, and writes it in decimal format — so it shouldn't change anything. But I'll see if there are other formats (like unit suffixes) that k8s allows, and add tests for them if so.

Reply:
I ended up going a different route: I just create some yaml, and then k8s's yaml library to turn it into the ResourceRequirements.